### PR TITLE
Upgrade versions

### PR DESCRIPTION
**Pull request by JoshLozensky**
Upgrading versions in files identified by dependabot.

### CODE REVIEW

**Review by msbw2 — APPROVED** [review]

File: src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Microsoft.AspNet.Mvc" Version="5.2.7" />
     <PackageReference Include="Microsoft.AspNet.Web.Optimization" Version="1.1.3" />
     <PackageReference Include="Microsoft.AspNet.WebApi" Version="5.2.7" />
-    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="5.5.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="8.1.2" />
```
Comment (jennyf19) [inline src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj:36]: web.config also has to be updated. Owin is a bit more complex. You will need to let VS Help and double click on the warning that should appear in the IDE.

Comment (kellyyangsong) [inline src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj:36]: Is this a breaking change if Owin customers were previously depending on `5.5.0`?

Comment (jennyf19) [reply src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj:36]: I'll punt to @jmprieur I think we've been leaving it at this version intentionally, not remembering why though.

**Review by jmprieur — APPROVED** [review]
LGTM except the changes to Microsoft.IdentityModel.Protocols.WsFederation version (we don't need that)

We should update the .dependabot config file to exclude Microsoft.IdentityModel.Protocols.WsFederation actually

File: src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj (hunk 1 above)
Comment (JoshLozensky) [reply src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj:36]: Synced with JM offline, he agrees that we should not update this, and believes we don't actually use it.

**Comment by JoshLozensky** [comment]
> We should update the .dependabot config file to exclude Microsoft.IdentityModel.Protocols.WsFederation actually

I'll do this in another PR

File: src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj (hunk 1 above)
Comment (JoshLozensky) [reply src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj]: We actually aren't taking the WsFederation change

**Comment by JoshLozensky** [comment]
> LGTM except the changes to Microsoft.IdentityModel.Protocols.WsFederation version (we don't need that)
> 
> We should update the .dependabot config file to exclude Microsoft.IdentityModel.Protocols.WsFederation actually

Updated the config file

**Review by jmprieur — APPROVED** [review]
Thanks @JoshLozensky 
#productivity

**Review by kellyyangsong — APPROVED** [review]
Thank you, Josh!

File: src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj (hunk 1 above)
Comment (kellyyangsong) [reply src/Microsoft.Identity.Web.OWIN/Microsoft.Identity.Web.OWIN.csproj]: Thanks for updating us here!!!! It made it so easy for me to glance at this thread and ascertain what we decided on/what happened.

**Review by jmprieur — APPROVED** [review]

**Pull request merged**